tools/wptserve/wptserve/server.py

```diff
@@ -727,3 +844,56 @@ def get_url(self, path="/", query=None, fragment=None):
         return urlunsplit(("http" if not self.use_ssl else "https",
                            "%s:%s" % (self.host, self.port),
                            path, query, fragment))
+
+
+class _WebSocketConnection(object):
+    """Mimic mod_python mp_conn."""
```

Comment:
The docstring setup here is very odd. I think it's better to have the class-level description in the `__init__` docstring rather than having a generic description of what constructing the class does there.